src/templates/partials/lighttpd.hbs

```diff
@@ -1,7 +1,11 @@
 # {{output.header}}
 # {{{output.link}}}
+$SERVER["socket"] ==     ":80" { }
```

Comment:
This only does IPv4?

Reply:
Yes, it only does IPv4.
Also, since the default is `server.port = 80`, I'm going to push a change to comment this line out.